schema.sql

```diff
@@ -0,0 +1,11 @@
+/* Database schema to keep the structure of entire database. */
+CREATE DATABASE vet_clinic;
+
+CREATE TABLE animals (
+    id INTEGER,
+    name VARCHAR(50),
```

Comment:
\[OPTIONAL\] I advise that you to make the `id` column not null and auto-incremented.

```sql
id int NOT NULL IDENTITY(1,1)
```

[AUTO INCREMENT Field](https://www.w3schools.com/sql/sql_autoincrement.asp)
Auto-increment allows a unique number to be generated automatically when a new record is inserted into a table.